indico/core/signals/event/designer.py

```diff
@@ -15,3 +15,9 @@
 The registration form is passed in the `regform` kwarg. The list of registration
 objects are passed in the `registrations` kwarg and it may be modified.
 ''')
+
+customize_badge_style = _signals.signal('customize-badge-style', '''
+Called when printing a badge.
+Each `item` in a badge and it's `styles` are passed in the kwarg. The `styles` is a dictionary
+which may be modified and is used to update the `item style`.
```

Comment:
This is a bit misleading; while modifying the dict would in fact work, the way the signal is being called, it should rather return a dict containing just the updates...